server/questionValidator.js

```diff
@@ -20,12 +20,21 @@ const common = require('./common.js');
 
 const successMsg = {success:true, err:'Validation Passed'}
 const failMsg = {success:false, err:common.getError(3022)}
-/*Send back specific error message by question type*/
+
+/**
+ * Send back specific error message by question type
+ *
+ * @param {string} message
+ */
 var qTypeFailMsg = function(message) {
     return {success:false, err:message};
 }
 
-/*Validate all question fields on first entry to db*/
+/**
+ * Validate all question fields on first entry to db
+ *
+ * @param {*} info
```

Comment:
Add type instead of `*`

Reply:
There is no type for this, its any type that is passed in, there is no one type. Unless I am missing something